src/main/java/com/plaid/client/response/IdentityGetResponse.java

```diff
@@ -3,77 +3,56 @@
 import java.util.List;
 
 public final class IdentityGetResponse extends BaseResponse {
-  private List<Account> accounts;
   private ItemStatus item;
-  private Identity identity;
+  private List<AccountWithOwners> accounts;
 
   public ItemStatus getItem() {
     return item;
   }
 
-  public Identity getIdentity() {
-    return identity;
+  public List<AccountWithOwners> getAccounts() {
+    return accounts;
   }
 
-  public List<Account> getAccounts() {
-    return accounts;
+  public static final class AccountWithOwners extends Account {
```

Comment:
I don't think this should be static, but I am not 100% sure.

@davidzhanghp

Reply:
He is in china :(

Reply:
Also looking at the other responses, all of the classes are static

Reply:
👍 sounds good.

Reply:
TransactionsGetResponse.java Is this have any fields of  
1- checkNumber
2- routingNumber
3-accountNumber
  if yes 
please let me know